packages/maven/build.sh

```diff
@@ -20,7 +19,7 @@ termux_step_make_install() {
 	mkdir -p $TERMUX_PREFIX/opt
 	cp -a $TERMUX_PKG_SRCDIR $TERMUX_PREFIX/opt/maven/
 	# Symlink only starter scripts for Linux
-	ln -sfr $TERMUX_PREFIX/opt/maven/bin/mvn $TERMUX_PREFIX/bin/mvn
-	ln -sfr $TERMUX_PREFIX/opt/maven/bin/mvnDebug $TERMUX_PREFIX/bin/mvnDebug
-	ln -sfr $TERMUX_PREFIX/opt/maven/bin/mvnyjp $TERMUX_PREFIX/bin/mvnyjp
+        for i in mvn mvnDebug mvnyjp; do
+                ln -sfr $TERMUX_PREFIX/opt/maven/bin/$i $TERMUX_PREFIX/bin/$i
+        done
```

Comment:
Use tabs for indentation in Shell scripts, as per <a href="https://github.com/termux/termux-packages/wiki/Coding-guideline">Coding guideline</a>.

Reply:
Committed.